src/ispc.cpp

```diff
@@ -1347,6 +1347,13 @@ void Target::markFuncWithTargetAttr(llvm::Function *func) {
     }
 }
 
+bool Target::IsGenericTypeLayoutIndeterminate(llvm::Type *type) {
```

Comment:
Let's change existing instances of "(m_isa == Target::GENERIC && lGenericTypeLayoutIndeterminate(type) == true)" to IsGenericTypeLayoutIndeterminate.